Debounce/BufferedEventArgs.cs

```diff
@@ -0,0 +1,29 @@
+﻿// SPDX-FileCopyrightText: 2024 Frans van Dorsselaer
```

Comment:
I did not write this :)

Just make yourself the copyright holder of the file. The fact that it is MIT licensed is sufficient to be included into this project. The FileContributor is intended for *others* that later feel they have made significant changes.

(some for other files)

Reply:
All right, done.